index.html

```diff
@@ -1,81 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
-    <style>
-      body {
-     background-color: rgb(192, 214, 255);
-    }
 
-    @media screen and (min-width: 800px) {
-  body {
-    background-color: lightgreen;
-  }
-}
-  #welcome-section{
-    height: 100vh;
-    /* padding-top: 20vh; */
-    margin-top: 100;
-  }
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="./css/style.css">
+</head>
 
-  header {
-  position: fixed;
-  width: 100%;
-}
+<body>
 
-#navbar{
-      top:0;
-      height: 50px;
-      position: fixed;
-    }
-    </style>
-  </head>
-
-  <body>
-  
-    <header>
-      <nav id="navbar">
-        <ul class="navbar-nav ml-auto">
-          <li class="nav-item">
-            <a class="nav-link" href="#about">About</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#projects">Project</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#contact">Contact</a>
-          </li>
-        </ul>
-      </nav>
-    </header>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-    <!-- <h1>Hello World, we are software developers!</h1> -->
-    <script  src="js/index.js"></script>
-    <section id="welcome-section">
-      <div>
-        <h1>Hey we are Team Elio and Amrendra</h1>
-        <p>We are Web Developer</p>
-      </div>
-    </section>
-    <section id="projects"><div>
+  <header>
+    <h1>Hey we are Team Elio and Amrendra</h1>
+    <nav id="navbar">
+      <ul class="navbar-nav ml-auto">
+        <li class="nav-item">
+          <a class="nav-link" href="#about">About</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#projects">Project</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#contact">Contact</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <!-- <h1>Hello World, we are software developers!</h1> -->
+  <script src="js/index.js"></script>
+  <section id="welcome-section">
+    <div>
+      <h2>Welcome to Developer Page</h2>
+      <p>We are Web Developer</p>
+    </div>
+  </section>
+  <section id="projects">
+    <div>
       <h1 class="project-tile">Projects</h1>
       <p>List of projects</p>
       <ul>
         <li> <a href="https://github.com/amrendrakind/Microverse_Project_Test2"> Project 1</a></li>
         <li>Project 2</li>
         <li>Project 3</li>

```

Comment:
It is better to  mention link of projects that you have done any.

Reply:
- [ ] Please explain to the student why it is important to add the link and how that might affect the overall outlook of the site.